tools/docker/builder/Dockerfile

```diff
@@ -35,19 +38,9 @@ RUN set -xe; \
         ERLANG_SHA256="80e95df44cba03f9abefa54ab1452d97b9969a6d164792e827e59a1116377bbd" && ERLANG_OS="arm64"; \
         JDK_SHA256="2e3c19c1707205c6b90cc04b416e8d83078ed98417d5a69dce3cf7dc0d7cfbca" && JDK_OS="aarch64"; \
         JQ_SHA256="af986793a515d500ab2d35f8d2aecd656e764504b789b66d7e1a0b727a124c44" && JQ_OS="linux64"; \
+        MOLD_SHA256="24ea75ada4337c5f509ad2eecf1a82d6047ab58c415a78374dfa7e6d69b06737" && MOLD_OS="aarch64-linux"; \
```

Comment:
downloaded and validated https://github.com/rui314/mold/releases/download/v1.3.1/mold-1.3.1-aarch64-linux.tar.gz

```
sha256sum mold-1.3.1-aarch64-linux.tar.gz
24ea75ada4337c5f509ad2eecf1a82d6047ab58c415a78374dfa7e6d69b06737  mold-1.3.1-aarch64-linux.tar.gz